pkg/vex/vulnerability.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2023 The OpenVEX Authors
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package vex
+
+// Vulnerability is a struct that captures the vulnerability identifier and
+// its aliases. When defined, the ID field should be an IRI.
+type Vulnerability struct {
+	ID          string            `json:"@id,omitempty"`
+	Name        VulnerabilityID   `json:"name,omitempty"`
+	Description string            `json:"description,omitempty"`
+	Aliases     []VulnerabilityID `json:"aliases,omitempty"`
```

Comment:
nit: It'd be nice to have doc comments on these fields, too.

Reply:
added